tests/ignite/engine/test_create_supervised.py

```diff
@@ -172,6 +172,73 @@ def _test_create_mocked_supervised_trainer(
                             assert training_step_mock.called
 
 
+def _test_gradient_accumulation(
```

Comment:
Is it possible to update the test `_test_create_supervised_trainer` instead of introduce this new one which seems to be a copy/paste with a little modification ?

Reply:
Okay in that case should I parameterize over gradient_accumulation_steps like this?

```python
test_create_supervised_parameters=("gradient_accumulation_steps,loss,weight,bias",[(1,...),(3,...)])

def _test_create_supervised_trainer(gradient_accumulation_steps,loss,weight,bias,....):
... 

@pytest.mark.parametrize(*test_create_supervised_parameters)
def test_create_supervised_trainer_with_cpu(gradient_accumulation_steps, loss, weight, bias):
    _test_create_supervised_trainer(gradient_accumulation_steps, loss, weight, bias, trainer_device="cpu")
```

Reply:
Or just call it twice with ``gradient_accumulation_steps``  as 1 and not 1

Reply:
> Okay in that case should I parameterize over gradient_accumulation_steps like this?
> 
> ```python
> test_create_supervised_parameters=("gradient_accumulation_steps,loss,weight,bias",[(0,...),(1,...),(3,...)])
> 
> def _test_create_supervised_trainer(gradient_accumulation_steps,loss,weight,bias,....):
> ... 
> 
> @pytest.mark.parametrize(*test_create_supervised_parameters)
> def test_create_supervised_trainer_with_cpu(gradient_accumulation_steps, loss, weight, bias):
>     _test_create_supervised_trainer(gradient_accumulation_steps, loss, weight, bias, trainer_device="cpu")
> ```

Yes, `gradient_accumulation_steps`, `loss`, `weight`, and `bias` arguments should be added.